[iOS] UI::Camera broken with undefined symbols

With RubyMotion 6.2 on macOS Mojave 10.14.5, I can't build an app for iOS 12.4 that uses Flow (i.e., motion-flow 0.1.8) because the build errors out due to undefined symbols in UI::Camera:

```
$ motion create Hello
$ cd Hello
$ echo "gem 'motion-flow'" >> Gemfile
$ bundle exec rake
```

That fails with messages like:

```
Undefined symbols for architecture x86_64:
  "_AVCaptureSessionPresetHigh", referenced from:
      _MREP_AA6FC69D5A7541E5BC60EA59CA1D1A0F in camera.rb.o
```

and ends with:

```
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
rake aborted!
```

If I delete `flow/ui/cocoa/camera.rb` from the installed gem, then the app builds successfully.

--- a/lib/cocoa.rb
+++ b/lib/cocoa.rb
@@ -30,5 +30,5 @@ Motion::Project::App.setup do |app|
   end
   app.spec_files.delete_if { |path| path.start_with?('./spec/helpers/android') }
 
-  app.frameworks += ['SystemConfiguration', 'CoreLocation', 'AddressBookUI']
+  app.frameworks += ['SystemConfiguration', 'CoreLocation', 'AddressBookUI', 'AVFoundation']
 end

